vault/core.go

```diff
@@ -4205,6 +4208,21 @@ func (c *Core) Events() *eventbus.EventBus {
 	return c.events
 }
 
+func (c *Core) GetAPIRedirect(ctx context.Context, path string) (string, error) {
+	if c.apiRedirects == nil {
```

Comment:
nit: We aren't consistent on checking for c.apiRedirects being nil, I don't think we need to as we init that member within CreateCore?